go/exorelay/features/connect_retry.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Retrying
+
+  As a develper building Exosphere applications
+  I want want ExoRelay to retry establishing a connection
```

Comment:
Let's use the [job story](https://jtbd.info/replacing-the-user-story-with-the-job-story-af7cdee10c27) syntax:

> When running Exocom applications
I  want ExoRelays to reconnect after network failurs
So that my applications is resilient to Exocom outages.